src/core_landice/mode_forward/mpas_li_iceshelf_melt.F

```diff
@@ -1166,6 +1166,7 @@ end subroutine iceshelf_melt_ismip6
 
     subroutine grounded_face_melt_ismip6(domain, err)
 
+      use li_calving  
```

Comment:
This dependency should also be added to the Makefile.

Reply:
Added